cmake/Modules/FindTPLMKL.cmake

```diff
@@ -72,5 +75,6 @@ ELSE()
       HEADER_PATHS
         ${MKL_ROOT}/include
     )
+    SET(BLAS_LIBRARIES "mkl_intel_lp64;mkl_intel_thread;mkl_core;iomp5")
```

Comment:
This is typically installation dependent, it is not clear that you will have all these libraries on every machine depending on how MKL is installed. I think it would be preferable to have the user set these manually in the `CMAKE_CXX_FLAGS` rather than force them to install MKL in a specific way to comply with the expected libraries above?

Reply:
Anyway, we need to access `${BLAS_LIBRARIES}` in `CheckHostBlasReturnComplex.cmake`.
Since this file finds MKL, it indeed should summarize the results in a variable for reference in other places.